src/protocol.h

```diff
@@ -275,6 +275,9 @@ enum ServiceFlags : uint64_t {
     // collisions and other cases where nodes may be advertising a service they
     // do not actually support. Other service bits should be allocated via the
     // BIP process.
+
+    // NODE_BIP148 means the node enforces BIP 148's mandatory Segwit activation beginning August 1, 2017
+    NODE_BIP148 = (1 << 27),
```

Comment:
Shouldn't this require an addition to the BIP, or at the very least, a mail to bitcoin-dev as stated in the comment above it? Is choosing a temporary service bit a good idea?